src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/DisputantUpdateRequest.java

```diff
@@ -0,0 +1,46 @@
+package ca.bc.gov.open.jag.tco.oracledataapi.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import org.hibernate.annotations.Type;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table
+@Getter
+@Setter
+@NoArgsConstructor
+public class DisputantUpdateRequest extends Auditable<String> {
+
+	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
+	@Id
+	@GeneratedValue
+	private Long disputantUpdateRequestId;
+
+	@Column
+	@NotNull
+	private Long disputeId;
+
+	@Column(length = 3)
+	@Schema(nullable = false)
+	@Type(type = "ca.bc.gov.open.jag.tco.oracledataapi.model.ShortNamedEnumType")
+	@NotNull
+	private DisputantUpdateType updateType;
+
+	@Column(length = 1000)
+	@Schema(nullable = false)

```

Comment:
PLease add an update status field with possible values Pending/Accepted, Rejected.  Instead of deleting from this queue, we will just update the status as per Adam Erickson

Reply:
Done.